### PR TITLE
Irma

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

File: data/gaTable.yaml [hunk 1]
```diff
@@ -999,7 +999,26 @@
...
-  projectContact: 'Jordan Walker'
-  projectContactEmail: 'jiwalker@usgs.gov'
+  projectContact: 'Jordan Read'
+  projectContactEmail: 'jread@usgs.gov'
+  analyticsContact: 'David Watkins'
+  analyticsContactEmail: 'wwatkins@usgs.gov'
+- longName: "VizLab: Hurricane Irma's Water Footprint"
+  shortName: VizLabIrma
+  accountName: Vizlab
+  viewID: '159805194'
```
[the remark this answers is not in the document]
Comment (wdwatkins) [reply data/gaTable.yaml]: (authenticate the same way the fetcher does)

[the remark this answers is not in the document]
Comment (ldecicco-USGS) [reply data/gaTable.yaml]: Is that wrong? I got it from https://ga-dev-tools.appspot.com/account-explorer/

[the remark this answers is not in the document]
Comment (wdwatkins) [reply data/gaTable.yaml]: Oh sorry, looks like I commented on the wrong line... I mean `internalWebPropertyId`

[the remark this answers is not in the document]
Comment (wdwatkins) [reply data/gaTable.yaml]: It shouldn't be the same as Harvey

[the remark this answers is not in the document]
Comment (ldecicco-USGS) [reply data/gaTable.yaml]: Any idea what the "profileId" is?
```
googleAnalyticsR::ga_view(accountId = '78530187', webPropertyId = 'UA-78530187-8', profileId = ???)
```
(and for the record....`ga_view` is going to return `internalWebPropertyId`?)

[the remark this answers is not in the document]
Comment (ldecicco-USGS) [reply data/gaTable.yaml]: Ah, I see it, it's the viewID

**Pull request merged**